addons/html_builder/static/src/website_preview/website_builder_action.js

```diff
@@ -64,10 +64,6 @@ export class WebsiteBuilder extends Component {
         this.setIframeLoaded();
         this.addSystrayItems();
         onWillDestroy(() => {
-            window.parent.document.removeEventListener(
-                "website_edit_loaded",
-                this.updateEditInteraction
-            );
```

Comment:
I suppose it's an oversight; `website_edit_loaded` is never added as an event listener